src/sdk/pynni/nni/metis_tuner/metis_tuner.py

```diff
@@ -392,6 +406,9 @@ def _selection(self, samples_x, samples_y_aggregation, samples_y,
                                   'expected_mu': expected_mu, 'expected_sigma': expected_sigma}
 
         outputs = self._pack_output(lm_current['hyperparameter'])
+        if outputs in self.history_parameters:
```

Comment:
could you add some logic about "take next config as exploration step randomly."

Reply:
OK~